WooCommerce/WooCommerceTests/ViewRelated/Orders/Order Details/OrderDetailsDataSourceTests.swift

```diff
@@ -569,6 +570,78 @@ final class OrderDetailsDataSourceTests: XCTestCase {
         }
     }
 
+    func test_WCShip_installation_section_is_not_visible_for_ineligible_order() throws {
```

Comment:
What do you think about renaming this method into `test_WCShip_installation_section_is_not_visible_when_WCShip_plugin_is_installed_and_active`?

I think this will reflect the exact scenario that we are testing in this test. 

It took me a while to understand why this order is "eligible". 😅